volkswagencarnet/vw_vehicle.py

```diff
@@ -185,6 +185,10 @@ async def update(self):
                 #     self.get_timerprogramming(),
                 #     return_exceptions=True,
             )
+            await asyncio.sleep(10)
+            await asyncio.gather(
+                self.get_service_status()
+            )
```

Comment:
I had to do it this way, otherwise `get_service_status` was executed before other APIs finished running.

Reply:
I don't like this 10-second delay.
I need to find a way to refresh the status separately from other sensors.

@oliverrahner if you have an idea how to do it, please let me know.
Will put it as a draft for the time being.

Reply:
not relevant anymore. can be ignored.